parlai/crowdsourcing/tasks/pairwise_per_turn_eval/analysis/compile_results.py

```diff
@@ -0,0 +1,553 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import re
+from collections import defaultdict
+from datetime import datetime
+from typing import Any, Dict
+
+import pandas as pd
+
+import parlai.utils.logging as logging  # TODO: check if this is how this import is usually done
+from parlai.crowdsourcing.utils.acceptability import AcceptabilityChecker
+from parlai.crowdsourcing.utils.analysis import AbstractResultsCompiler
+from parlai.crowdsourcing.tasks.pairwise_per_turn_eval.per_turn_eval_blueprint import (
+    BLUEPRINT_TYPE,
+)  # noqa: F401  # For registering the blueprint
```

Comment:
Jack just suggested a fix for this linting error: see https://github.com/facebookresearch/ParlAI/pull/4333/files#r794681652